packages/contentstack/src/hooks/prerun/latest-version-warning.ts

```diff
@@ -0,0 +1,60 @@
+import { cliux, configHandler, HttpClient, LoggerService } from '@contentstack/cli-utilities';
+import * as semver from 'semver';
+interface ICacheData {
+  lastChecked: number;
+  lastWarnedDate: string;
+  latestVersion: string;
+}
+interface IVersionUpgradeWarningFrequency {
+  versionSyncDuration: number;
+}
+const versionUpgradeWarningFrequency: IVersionUpgradeWarningFrequency = {
+  versionSyncDuration: 3 * 24 * 60 * 60 * 1000,
+};
+export default async function (_opts): Promise<void> {
+  const now = Date.now();
+  const today = new Date().toISOString().split('T')[0];
+  let logger!: LoggerService;
+  logger = new LoggerService(process.cwd(), 'cli-log');
```

Comment:
@naman-contentstack , can we declare and initialize in same line?

Reply:
Done